services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -637,6 +637,20 @@ export const deleteAllActions = (): void => {
     UpdateAction.set({ actions: [] });
 };
 
+export const useDeleteEnvironmentsForAppActions = (app: string): string[] => {
```

Comment:
nitpick: maybe rename this
```suggestion
export const useDeleteEnvironmentActionsForApp = (app: string): string[] => {
```